README.md

```diff
@@ -12,6 +12,9 @@ yarn
 
 ```
 yarn start
+# or
+yarn serve
```

Comment:
`yarn serve` won't work without a `yarn build` first. It's a less good for development because you don't get the auto-rebuild feature.